package.json

```diff
@@ -22,30 +22,33 @@
     }
   ],
   "main": "lib/image-diff",
+  "files": [
+    "lib/"
```

Comment:
What's the purpose of files? Isn't the `.gitignore` good enough? Also, this might remove the ability to run tests for a dependency:

```bash
npm run <package> test
```